packages/rum-core/src/domain/rumSessionManager.ts

```diff
@@ -28,6 +29,7 @@ export const enum RumTrackingType {
   NOT_TRACKED = '0',
   TRACKED_WITH_SESSION_REPLAY = '1',
   TRACKED_WITHOUT_SESSION_REPLAY = '2',
+  TRACKED_WITH_FORCED_REPLAY = '3',
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** could you support this new state in the extension?
‏
![2024-04-24_11-57-42 (1)](https://github.com/DataDog/browser-sdk/assets/1331991/0e83ed8e-8f4a-48d3-86e8-9c4018decb1f)

Reply:
Done ✔️